Documentation/installation/k8s-install-migration.rst

```diff
@@ -193,6 +193,34 @@ Preparation
         $ cilium install --helm-values values-migration.yaml --helm-auto-gen-values values-initial.yaml
         $ cat values-initial.yaml
 
+.. note::
+In case you are migrating from Calico you need to ensure that Calico doesn't accidently switch Cilium's interfaces during the migration.
+
+Calico has several modes of operation to decide which network interface to use by default on nodes that it manages. This is configured using the Tigera Operator's ``spec.calicoNetwork.nodeAddressAutodetectionV4`` (and respectively ``nodeAddressAutodetectionV6`` for IPv6) parameter.
```

Comment:
```suggestion
For example, Calico has several modes of operation to decide which network interface to use by default on nodes that it manages. This is configured using the Tigera Operator's ``spec.calicoNetwork.nodeAddressAutodetectionV4`` (and respectively ``nodeAddressAutodetectionV6`` for IPv6) parameter.
```

Reply:
+1 to @lizrice. The only thing I would change is to be more specific: "This is configured" to "Operation mode is configured"